tests/testdata/plugins/test-plugin-with-errors/i18-usage-error-file.php

```diff
@@ -5,11 +5,11 @@
 
 $city = 'Surat';
 
-printf(
+sprintf(
```

Comment:
It would be useful to have an inline comment above each of these errors to explain what error(s) they are trying to demonstrate.

Reply:
Thanks @joemcgill I'm updating these test plugins in #17 I'll add comments in that PR.